src/masternodes/threadpool.h

```diff
@@ -19,7 +19,7 @@ class TaskPool {
     public:
         explicit TaskPool(size_t size);
         void Shutdown();
-        [[nodiscard]] size_t GetAvailableThreads() const { return size; }
+        size_t GetAvailableThreads() { return size; }
```

Comment:
Why were the attributes removed ?

Reply:
`const` is removed because this is a more abstract API. Will likely not be const later as it will return dynamic value of threads. Can probalby add nodiscard.